docker/build-tools/Dockerfile

```diff
@@ -57,30 +62,33 @@ RUN go get github.com/jteeuwen/go-bindata/go-bindata
 RUN go get istio.io/tools/protoc-gen-docs
 RUN go get istio.io/tools/cmd/annotations_prep
 
-# Save some bytes in the container
-RUN upx --lzma /go/bin/*
+# Put the stuff we need in its final output location
+RUN mkdir -p ${OUTDIR}/usr/bin
+RUN mkdir -p ${OUTDIR}/usr/include
+RUN mkdir -p ${OUTDIR}/usr/local/go/bin
+RUN mkdir -p ${OUTDIR}/usr/local/go/pkg/mod
```

Comment:
Is this needed? I think when the container is invoked, this path ends up being mounted to point to the host's environment, so it doesn't need to exist in the container's file system.

Reply:
i cleaned htis up . a bit.